src/index.js

```diff
@@ -9,6 +9,7 @@ import Bookings from './Bookings'
 // import './images/turing-logo.png'
 let globalRooms = null;
 let globalBookings = null;
+let globalUsers = null;
```

Comment:
So I'm thinking back to a comment you left on one of my PRs about getting an undefined global variable to hold the data fetched from the API. I'm guessing these nulls are what you were talking about. I might have to look at the rest of your code, but my first guess is that I ran each set of data retrieved from each fetch through a map to create the class instances I needed and then pushed the new arrays from those maps up to the globals. Maybe that's why they held? Not sure, but that would be my first guess 🤷‍♀️